client/core/core.go

```diff
@@ -6847,6 +6847,12 @@ func handleRevokeOrderMsg(c *Core, dc *dexConnection, msg *msgjson.Message) erro
 		return fmt.Errorf("no order found with id %s", oid.String())
 	}
 
+	if tracker.status() == order.OrderStatusRevoked {
+		// Already revoked is expected if entire book was purged in a suspend
+		// ntfn, which emits a gentler and more informative notification.
+		// However, we may not be subscribed to orderbook notifications.
+		return nil
```

Comment:
I'd also note that that this is a sensible check anyway to silence error messages from `tracker.revoke()` on account of the myriad of other sefl-revoke causes.